docs/v8-migration-api.md

```diff
@@ -77,6 +77,46 @@ Check out the [v8-migration guide](v8-migration.md) for other user-level changes
     - For `node.parameters` in Babel 7, use `node.params` in Babel 8
     - For `node.typeAnnotation` in Babel 7, use `node.returnType` in Babel 8
 
+- Wrap `TSEnumMember` within the `TSEnumBody` node ([#16979](https://github.com/babel/babel/pull/16979))
+
+  In Babel 8, all members of a `TSEnumDeclaration` will be wrapped within a `TSEnumBody` node, which can be accessed from the `.body` property.
+
+  ```ts
+  // Example input
+  enum ColorType {
+    Red,
+    Green,
+    Blue,
+  }
+
+  // AST in Babel 7
+  {
+    type: "EnumDeclaration",
```

Comment:
```suggestion
    type: "TSEnumDeclaration",
```

Reply:
Thanks! I should have double checked my ASTExplorer setting.